README.md

```diff
@@ -328,6 +328,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Eclipse](http://www.eclipse.org/) - Established, open-souce project with support for lots of plugins and languages.
 * [IntelliJ IDEA](http://www.jetbrains.com/idea/) - Supports a lot of JVM languages and provides good options for Android development. The commercial edition targets the enterprise sector.
 * [NetBeans](https://netbeans.org/) - Provides integration for several Java SE and EE features from database access to HTML5.
+* [BlueJ](http://www.bluej.org//) - A free Java Development Environment designed for beginners, used by millions worldwide.
```

Comment:
I'm against recommending this dummy IDE. There is no advantage using it for teaching Java. I would even argue that it leads to false conceptions of programming in general.